content/docs/user-guide/how-to/best-practices.md

```diff
@@ -0,0 +1,131 @@
+# Best Practices for DVC Projects
+
+DVC provides a systematic approach towards managing and collaborating on data
+science projects. You can manage your projects with DVC more efficiently using
+the practices listed here:
```

Comment:
This file should be removed from here. That's the point of extracting this to #1748.